### DISCUSSION TITLE
SISTEM ROUTER NO WORKING

### DISCUSSION THREAD

**Discussion by JonathanFerreirar** [vercel/community#2908] [Help] [answered]
### Summary

I have a site in react and the route system is not working, I have already deployed other sites using the route system and I never had a problem here is my web-site if anyone can help me thanks.

### Example

https://techlong.vercel.app/

### Steps to Reproduce

just try go to any other page.

**Comment by amyegan** [comment]
It looks like you found a solution! The router seems to be working now when I visit the site

**Comment by JonathanFerreirar** [comment, marked as answer]
Yes i'm found, thanks.😊

On Wed, Jun 21, 2023 at 5:49 PM Amy ***@***.***> wrote:

> It looks like you found a solution! The router seems to be working now
> when I visit the site
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/2908#discussioncomment-6245935>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZQM3EPNHI2AQJDDAPGXHI3XMNM5DANCNFSM6AAAAAAZLE74XM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

      Jonathan

*     (31) 3222-9924 – **ramal 200 *

*                Contec - Contabilidade Técnica Eireli*



                 ***@***.***
